tests/src/FunctionalJavascript/MediaAvPortalCreateContentTest.php

```diff
@@ -164,20 +163,28 @@ public function testAvPortalPhotoMediaEntity(): void {
     $this->assertContains('ec.europa.eu/avservices/avs/files/video6/repository/prod/photo/store/', $image_url);
     $this->assertContains('P038924-352937.jpg', $image_url);
 
-    // Edit the newly created media.
-    $this->drupalGet('media/1/edit');
+    $photo_urls = [
+      'https://audiovisual.ec.europa.eu/en/album/M-090909/P-039162~2F00-12',
+      'https://audiovisual.ec.europa.eu/en/photo/P-039162~2F00-12',
+    ];
 
-    // Update the field.
-    $page->fillField('Media AV Portal Photo', 'https://audiovisual.ec.europa.eu/en/photo/P-039162~2F00-12');
-    $page->pressButton('Save');
+    // We could update photo with different URL formats supporting.
+    foreach ($photo_urls as $photo_url) {
+      // Edit the newly created media.
+      $this->drupalGet('media/1/edit');
 
-    // Visit the updated media content.
-    $page->clickLink('Andrus Ansip Vice-President of the EC addresses the Plenary of the European Parliament on the beginning of the Romanian Presidency of the Council of the EU');
+      // Update the field with url from album.
```

Comment:
We test both album and alone photos, so no need to change this comment from "Update the field."